src/_gettsim_tests/_policy_test_utils.py

```diff
@@ -29,22 +29,34 @@ def merged_input_df(self) -> pd.DataFrame:
     def merged_output_df(self) -> pd.DataFrame:
         return pd.concat([test.output_df for test in self.test_data], ignore_index=True)
 
+    def filter_test_data(
+        self, *, test_name: str | None = None, date: datetime.date | None = None
+    ) -> PolicyTestSet:
```

Comment:
I think a docstring with an example would be very helpful here, many of our core contributors won't be too familiar with type hints, wildcards, etc.

Reply:
I've added a docstring.